### PR TITLE
Add an integration test for parts with filesets

### PR DESCRIPTION
**Pull request by jocave**
Use a part from the parts service which uses filesets

### CODE REVIEW

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

File: integration_tests/test_parts.py [hunk 1]
```diff
@@ -61,3 +61,39 @@ def test_curl_exists_and_properly_defined(self):
...
+
+
+class PartsWithFilesetsTestCase(integration_tests.TestCase):
+
+    def test_part_with_fileset(self):
+        self.run_snapcraft('update')
+
+        output = self.run_snapcraft(['define', 'simple-make-filesets'])
+
+        expected = (
```
Comment (sergiusens) [inline integration_tests/test_parts.py]: for easier reading might consider using `"""` here?
Like
```python
    expected = ("""Maintainer: ...
Description: ...
...
""")
```

**Comment by jocave** [comment]
@sergiusens Correct - I didn't add the part there as from what I could see the integration tests were getting their parts from the real server. Is the idea to have the part in fake_servers as well even though this test wouldn't use it?

Sorry for not having a fix for the functionality that this test exercises. I had a look at the parts code today, but found it tricky to follow.

**Comment by sergiusens** [comment]
El viernes, 26 de agosto de 2016 10h'01:47 ART, Jonathan Cave 
<notifications@github.com> escribió:
> @sergiusens Correct - I didn't add the part there as from what 
> I could see the integration tests were getting their parts from 
> the real server. Is the idea to have the part in fake_servers as 
> well even though this test wouldn't use it?

You are indeed correct, I was thinking about the parts parser service which 
is all faked.

> Sorry for not having a fix for the functionality that this test 
> exercises. I had a look at the parts code today, but found it 
> tricky to follow.

That's fine, the parts code and lifecycle are very complex in themselves. 
There is a much needed refactor needed there, Joe has worked on it for a 
bit but it is still some ways ahead.


-- 
Enviado con Dekko desde mi dispositivo Ubuntu

File: integration_tests/test_parts.py (hunk 1 above)
Comment (sergiusens) [inline integration_tests/test_parts.py]: nevermind this comment, the rest of this file has this format

**Comment by sergiusens** [comment]
OK, revisiting this, I think `define` is fine an what needs fixing is the use of the fileset during execution

**Comment by sergiusens** [comment]
err, ignore me

**Comment by sergiusens** [comment]
#799 has this test in.
thanks

**Pull request closed without merging**